test/functional/test.bat

```diff
@@ -17,5 +17,31 @@ for %%a in (%sites%) do (
     )
 )
 
+set sites_convert=test_site_convert
+
+for %%a in (%sites_convert%) do (
+
+    echo(
+    echo Running %%a tests
+
+    node ../../index.js init %%a\non_markbind_site -c
+
+    node ../../index.js build %%a\non_markbind_site
+
+    xcopy %%a\non_markbind_site\_site %%a /i
+
+    node testUtil/test.js %%a
+
+    if errorlevel 1 (
+        echo Test %%a Failed
+        exit /b %errorlevel%
+    )
+
+    rmdir /s /q %%a\_site
```

Comment:
If test fails, generated files will left in the test site.

Reply:
I used `trap` in `test.sh`, but I couldn't find a batch equivalent. Is there a better fix than copying over the delete commands to the `if` block?

Reply:
I think you can set a variable and check after removing the directories. 

https://stackoverflow.com/questions/19300813/how-to-store-errorlevel-in-a-variable